osu.Game/Skinning/SkinProvidingContainer.cs

```diff
@@ -21,8 +23,10 @@ public class SkinProvidingContainer : Container, ISkinSource
     {
         public event Action SourceChanged;
 
-        [CanBeNull]
-        private readonly ISkin skin;
+        /// <summary>
+        /// The list of skins provided by this <see cref="SkinProvidingContainer"/>.
```

Comment:
I think this should probably mention that lookups happen sequentially on skins in this list.

Reply:
Also it should be reworded as this list seems to be intended to be used primarily to *add* new sources.

```suggestion
        /// Skins which should be exposed by this container, in order of lookup precedence.
```